File: pset/pset6/mario.py
import cs50


def main():

    height = cs50.get_int("Enter an integer between 1-8 inclusive: ")

    while height < 1 or height > 8:
        height = cs50.get_int("Re-enter valid height: ")

    space = height - 1

    for i in range(height):
        print(" " * space, end="")
        print("#" * (height - space), end="")
        print("  ", end="")
        print("#" * (height - space))
        space -= 1


main()

File: pset/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

void spaces(int n);
void hash(int n);

int main(void)
{
    int size = get_int("Input a number between 1-8 inclusive: ");
    int space = 0;

    while (size < 1 || size > 8)
    {
        size = get_int("Try again. Input a number between 1-8 inclusive: ");
    }

    // For loop for # of layers

    for (int i = 0; i < size; ++i)
    {
        space = size - i;

        spaces(space);

        hash(i);

        printf("  ");

        hash(i);

        printf("\n");
    }
}

void spaces(int space)
{
    for (int ii = 0; ii < (space - 1); ++ii)
    {
        printf(" ");
    }
}

void hash(int i)
{
    for (int ii = 0; ii < (i + 1); ++ii)
    {
        printf("#");
    }
}